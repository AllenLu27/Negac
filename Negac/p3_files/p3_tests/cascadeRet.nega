bool main(){
	return false && var;
	return var && false;
}
