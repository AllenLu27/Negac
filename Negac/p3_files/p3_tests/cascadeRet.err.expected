FATAL [2,18]: Undeclared identifier
FATAL [3,9]: Undeclared identifier
Semantic analysis failed
